packages/core/useElementHover/index.md

```diff
@@ -22,3 +24,25 @@ const myHoverableElement = ref()
 const isHovered = useElementHover(myHoverableElement)
 </script>
 ```
+### As a directive
+
+<LearnMoreComponents />
+
+```html
+<script setup lang="ts">
+import { ref } from 'vue'
+import { vHover } from '@vueuse/components'
+
+const isHovered = ref(false)
+function onHover(state: boolean) {
+  isHovered.value = state
+}
+</script>
+
+<template>
+  <button v-hover="onHover">
```

Comment:
I think we should match with the function name and call it vElementHover

Reply:
Fixed